index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page Built by Sheldon and Muda</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Nelson Mandela</h1>
+        <div id="img-div">
+            <img class="image" src="image/pexels-minenhle-shelembe-5985865.jpg" alt="Nelson Mandela">
+            <figcaption id="img-caption">Picture of Nelson Mandela</figcaption>
+        </div>
+        <div class="tribute-info" >
+            <p>About Nelson madela</p>
+            <a class="info" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank">Read More about our hero Nelson Mandela</a>
+        </div>
```

Comment:
For example, this is an inline comment 🙂 from lines `17` to `20`, it appears below the review. Also make sure that when you add one, you click on `Start a review` and not `Add single comment`. The difference? 🤔 If you click on `Add single comment`, the comment will appear before the review and seem unrelated to it, but if you click on `Start a review`, it only gets posted once you submit the review and appears below it. I hope this helps, let me know if you have any questions ❓ 😉